app/views/projects/edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<div>
+  <h1>Edit Project</h1>
+
+
+  <%= form_with(model: @project, local: true, url: change_project_path) do |form|  %>
```

Comment:
Suggest replacing all form blocks with `simple_form_for` ([example](https://github.com/TechforgoodCAST/fusebox-insights/blob/master/app/views/unknowns/_form.html.erb)) - as you'll get styles for inputs and errors displayed for free. [This way](https://github.com/TechforgoodCAST/fusebox-insights/tree/master/app/views/unknowns) of organising view files might also be of interest.